github-event/handler.go

```diff
@@ -168,7 +168,8 @@ func garbageCollect(garbageRequests []GarbageRequest) error {
 		if res.Body != nil {
 			defer res.Body.Close()
 		}
-		if res.StatusCode != http.StatusOK {
+		fmt.Printf("Status code for garbageCollect: %d\n", res.StatusCode)
```

Comment:
Only `log.Printf` if this is not the desired status code?

Reply:
This the status code for every request printed since you had comment above that its good idea to print all the status codes to check whats up with every request `Perhaps we should also include the status code in the message - this would have helped you in this instance.`  I think that's what you meant.